### DISCUSSION TITLE
Built-in Badware risks disable Context-menu Copy (greyed out) on Firefox 121

### DISCUSSION THREAD

**Discussion by quimnuss** [uBlockOrigin/uAssets#21700] [General]
### Prerequisites

- [X] This is **NOT** a [YouTube](https://www.github.com/uBlockOrigin/uAssets/issues/20586), [Facebook](https://www.github.com/uBlockOrigin/uAssets/issues/3367) or [Twitch](https://www.github.com/uBlockOrigin/uAssets/issues/5184) report.
- [X] I read and understand the [policy about what is a valid filter issue](https://github.com/uBlockOrigin/uAssets/blob/master/README.md#uassets).
- [X] I verified that this issue is not a duplicate. (Search [here](https://github.com/uBlockOrigin/uAssets/issues) to find out.)
- [X] I did not remove any of the [default filter lists](https://github.com/uBlockOrigin/uAssets/assets/66517106/6e8d024a-5869-47cf-b142-b2357eb6c7d0), or I have verified that the issue was not caused by removing any of the default lists.
- [X] I did not enable additional filter lists, or I have verified that the issue still occurs without enabling additional filter lists.
- [X] I do not have custom filters/rules, or I have verified that the issue still occurs without custom filters/rules.
- [X] I am not using uBlock Origin (uBO) along with other content blocker extensions.
- [X] I have verified that the web browser's built-in blocker or [DNS blocking](https://en.wikipedia.org/wiki/DNS_blocking) (standalone or through a VPN) is not causing the issue.
- [ ] I did not answer truthfully to **ALL** the above checkpoints.

### URL(s) where the issue occurs.

It's not very relevant. Google something, open it and right-click. Copy will be greyed out.

### Description

See this mozilla bug report https://bugzilla.mozilla.org/show_bug.cgi?id=1863246
It's actually caused by ublock Origin builtins - Badware risks

If you disable that builtin default filter, you solve this context menu copy greyed out behaviour.

quote of the issue:

I can reproduce this reliably by googling something, opening a result and without switching focus, right-clicking. Firefox 121.0 (64-bit) on linux Mint

Can confirm that disabling ublock origin fixes it
Can confirm that the addon https://addons.mozilla.org/en-US/firefox/addon/absolute-enable-right-click/ fixes it
Can confirm switching focus fixes it

I've narrowed it down to ublock origin -> Settings -> Build-ins -> Badware Risks

So I think it's not a Firefox issue but probably an uBlock Origin one. I'll report if I experience it again with those filters disabled.

### Other extensions used

none

### Screenshot(s)

<details>
<summary>Screenshot(s)</summary>

</details>

### Configuration

<details>
<summary>Details</summary>

```yaml
uBlock Origin: 1.54.0
Firefox: 121
filterset (summary):
 network: 223904
 cosmetic: 248852
 scriptlet: 48467
 html: 2183
listset (total-discarded, last-updated):
 removed:
  ublock-badware: null
 added:
  adguard-generic: 81578-5770, 21h.1m
  adguard-mobile: 8950-142, 21h
  adguard-social: 22155-876, 10m
  fanboy-thirdparty_social: 69-0, 10m
  fanboy-cookiemonster: 49207-3789, 10m
  fanboy-social: 16032-25, 10m
  ublock-annoyances: 6413-877, 10m
  dpollock-0: 11620-464, 10m
  spa-0: 3774-3, 7h.13m
  spa-1: 5692-159, 21h.1m
  adguard-spyware-url: 1378-92, 21h.1m
  block-lan: 55-0, 20d.1h.31m
  [12 lists not shown]: [too many]
 default:
  user-filters: 1-0, never
  ublock-filters: 37019-599, 9m
  ublock-privacy: 950-0, 9m
  ublock-unbreak: 2219-2, 9m
  easylist: 76467-895, 1h.17m
  easyprivacy: 33178-1003, 1h.17m
  urlhaus-1: 9865-0, 7h.13m
  plowe-0: 3784-1, 10m
  ublock-quick-fixes: 169-0, 9m
filterset (user): [array of 1 redacted]
userSettings:
 advancedUserEnabled: true
hiddenSettings:
 userResourcesLocation: [redacted]
supportStats:
 allReadyAfter: 430 ms (selfie)
 maxAssetCacheWait: 222 ms
popupPanel:
 blocked: 23
 network:
  github.com: 23
```
</details>

**Comment by ghost** [comment]
Not uBO:

https://bugzilla.mozilla.org/show_bug.cgi?id=1863246#c25  
https://bugzilla.mozilla.org/show_bug.cgi?id=1863246#c37
